docs/axes/cartesian/time.md

```diff
@@ -161,6 +161,35 @@ let chart = new Chart(ctx, {
 });
 ```
 
+## Changing the scale type from Time scale to Logarithmic/Linear scale.
+
+When changing the scale type from Time scale to Logarithmic/Linear scale, you need to add `bounds: ticks` to the scale options. Changing the `bounds` parameter is necessary because its default value is the `date` for the Time scale.
```

Comment:
```suggestion
When changing the scale type from Time scale to Logarithmic/Linear scale, you need to add `bounds: 'ticks'` to the scale options. Changing the `bounds` parameter is necessary because its default value is the `'date'` for the Time scale.
```

Reply:
fixed

Reply:
@dangreen fixed